api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentService.java

```diff
@@ -122,14 +103,14 @@ public Payment makeFirstPayment(PaymentCycle paymentCycle, String cardAccountId)
      * @return The {@link Payment} entity relevant to this process.
      */
     @SuppressWarnings("PMD.NullAssignment")
-    public Payment makePayment(PaymentCycle paymentCycle, String cardAccountId) {
+    public Payment makeRegularPayment(PaymentCycle paymentCycle, String cardAccountId) {
```

Comment:
In discussion with Matt and Gethyn, `makePaymentForCycle` fits better.

Reply:
Changed to makePaymentForCycle